src/config/reader.rs

```diff
@@ -213,10 +214,11 @@ impl ConfigReader {
         Ok(config_module)
     }
 
-    /// Checks if path is absolute else it joins file path with relative dir
-    /// path
+    /// checks and returns valid path for caller function.
```

Comment:
kept most of contents same and extended for urls, please let me know if this works for you or not.

```
   /// Checks if path is a URL or absolute path, returns directly if so.
    /// Otherwise, it joins file path with relative dir path.
```